components/dashboard/src/admin/UserDetail.tsx

```diff
@@ -104,45 +104,52 @@ export default function UserDetail(p: { user: User }) {
                     <div className="flex w-full mt-6">
                         <Property name="Sign Up Date">{moment(user.creationDate).format('MMM D, YYYY')}</Property>
                         <Property name="Remaining Hours"
-                            action={
-                                accountStatement && {
+                            actions={
+                                accountStatement && [{
                                     label: 'View Account Statement',
                                     onClick: () => setViewAccountStatement(true)
-                                }
+                                }, {
+                                    label: 'Grant 20h',
```

Comment:
**question**: This works fine as is. Just adding an alternative phrase in case this seems better. Feel free to leave this as is.

```suggestion
                                    label: 'Add 20 additional hours',
```

Reply:
Makes sense, thanks! I've slightly edited your suggestion so that I can apply it in one click 😇 (notes: I prefer "Grant" over "Add" because I find it more explicit + capitalized like other action labels + "Extra" is shorter than "Additional" 🚀)